exercises/pascals-triangle/description.md

```diff
@@ -4,10 +4,11 @@ In Pascal's Triangle each number is computed by adding the numbers to
 the right and left of the current position in the previous row.
 
 ```text
-    1
-   1 1
-  1 2 1
- 1 3 3 1
-1 4 6 4 1
+      1
+     1 1
+    1 2 1
+   1 3 3 1
+  1 4 6 4 1
+1 5 10 10 5 1
```

Comment:
when there were only single-digit numbers, it was the case that no number is directly above another.

but right now, there is a 1 directly above each 5, and there is a 4 directly above each 10

so this example is not currently showing the nature of the triangle

It is possible that this could be fixed. If it were, that would be better.

some possible solutions involve an unequal number of spaces in each gap in the same row. the resulting inconsistency raises questions about whether the relationship is different simply because of the difference in representation.

If a sixth row is to be added, such concerns should be addressed

Reply:
I agree that adding the line makes reading the triangle harder, while not providing much benefit I feel. @hekrause would you be opposed to removing the extra line?

Reply:
Ok, I reset the description in the exercism/problem-specification repository. Can we change the description in this repository automatic or do i have to make the same changes in this repository by hand?